metadata-ingestion/examples/recipes/sagemaker_to_datahub.yml

```diff
@@ -0,0 +1,9 @@
+source:
+  type: sagemaker
+  config:
+    aws_region: "us-west-2"
```

Comment:
might want to make it clear that AWS creds are autodetected - just a comment would be enough